### PR TITLE
JNG-4989 update jsl documentation

### PR DESCRIPTION
**Pull request by gprivi**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4989" title="JNG-4989" target="_blank">StoryJNG-4989</a>  Update JSL documentation
  </td></table>
  <br />
 

JNG-4989 update jsl documentation

### CODE REVIEW

File: judo-runtime-core-jsl-itest/models/pom.xml [hunk 1]
```diff
@@ -43,7 +40,7 @@
         <module>TransferAggregationModel</module>
         <module>TransferAssociationModel</module>
         <module>TransferConstructorModel</module>
-        <module>RecursiveCompositionModel</module>
+<!--        <module>RecursiveCompositionModel</module> -->
```
Comment (robertcsakany) [inline judo-runtime-core-jsl-itest/models/pom.xml]: Why is it removed?

**Review by robertcsakany — CHANGES_REQUESTED** [review]

**Review by Madmat8 — APPROVED** [review]

File: judo-runtime-core-jsl-itest/models/pom.xml (hunk 1 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/models/pom.xml]: Added back without automapped test

Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/models/pom.xml]: [75268a2](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/241/commits/75268a2d52d7d67a4551a6495568f3a0aae606f6)

Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/models/pom.xml]: [ebb9ce9](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/241/commits/ebb9ce9bb6dfd790c87e92d80b76f843e4a2a954)

**Review by bencegelei — APPROVED** [review]

**Review by bencegelei — CHANGES_REQUESTED** [review]
CI build failed

**Review by bencegelei — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Pull request merged**